### ISSUE TITLE
Add tests, and run Travis for PRs

### ISSUE THREAD

**Issue by junderw** [btcpayserver/node-btcpay#16]
After #15 is merged, we can run a Travis matrix that checks formatting, runs tslint, and runs tests.

Currently tests are lacking... Can be added in separate PR, I think #15 should be merged and released as a patch. (Current version has empty interface for Invoices, which is not helpful...)

**Comment by NicolasDorier** [comment]
Additional note: The tests can just depends on a shared account on https://testnet.demo.btcpayserver.org .

**Comment by takinbo** [comment]
#42 supersedes testing with a shared account and will close this when it gets merged.

**Comment by junderw** [comment]
Might want to use Github Actions since Travis is unreliable for OSS projects lately.

**Comment by junderw** [comment]
Closed in #55
